src/ctc/cctc-tombstone.cc

```diff
@@ -24,11 +24,45 @@
 namespace kaldi {
 namespace ctc {
 
+void SwapDimsForX(int32& xdim, int32& ydim, int32& zdim,
```

Comment:
sorry- it's against the Google style guide to use non-const references in function parameters.  These should be pointers.